src/Tests/Equinor.ProCoSys.DbView.WebApi.IntegrationTests/testappsettings.json

```diff
@@ -1,9 +1,9 @@
 {
   "Authority": "https://login.microsoftonline.com/3aa4a235-b6e2-48d5-9195-7fcf05b459b0",
-  "ApplicationUrl": "https://localhost:47501",
```

Comment:
talk about this?

Reply:
Moved local url to user secrets to enable execution in pipeline to use url from testappsettings.json. Should maybe use a better method to have multiple configurations...

Reply:
we can dive into that when creating prod stage